src/components/dropdown-disclosure/dropdown-disclosure.module.css

```diff
@@ -23,7 +23,7 @@
 	padding-top: 4px;
 	position: absolute;
 	width: fit-content;
-	z-index: 1;
+	z-index: 2;
```

Comment:
The disclosure was showing up behind the bookmark icon 🤷‍♀️

Reply:
Oh good catch! This is definitely the way. 👍🏻